### DISCUSSION TITLE
Tracking of track fats and protein

### DISCUSSION THREAD

**Discussion by inojuer** [NightscoutFoundation/xDrip#2872] [Q&A]
Hello all,

Having the possibility to track fats and protein would be most helpful in managing my bloodsugar and I have missed this feature a lot. 
Searching the database I found a post, describing how to implement this feature: https://github.com/NightscoutFoundation/xDrip/pull/2129

I'm absolutely not technically gifted... it's hard for me to even understand the "normal" posts, Technical posts like the above are unfortunately out of my reach.

I'm hoping that someone here might help me out?

How do I get the code for the "possibility to submit fat and protein treatments" into xDrip?

I will have to do in manually, right? Where and how? I only have the xDrip App and the users' interface and have no idea where to paste any code or even how to access the code at all? And - which of the many codes in the post above ist the one to use?

I'm using:
mobile: Pixel 6a
Android version 13
xDrip version e50dfdf-2023.05.03

I'm thankful for ANY help.
Thanks a lot and keep the great work!!!
Evi

**Comment by Navid200** [comment]
> ... and I have missed this feature a lot.  
  
Do you mean you have had this feature before?
In what app?

**Reply by inojuer** [reply]
what I meant was that I missed the feature from day one of using xDrip and am still missing it. I experience proteins and fats as a very important factor for blood sugar change. 
(english is not my mother tongue ;-)

I wouldn't know of any other app for diabetics with integrated tracking of fats and proteins.

Food tracking apps do have this feature of course, but then there`s no integration of blood sugar numbers and of insulin on board.
Cronometer (pro) can be synched with Dexcom for blood sugar values, but only if you have an U.S. Dexcom account which I don`t as I'm living in Germany.
